src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -4,26 +4,36 @@ import org.broadinstitute.dsde.firecloud.core.GetEntitiesWithType.EntityWithType
 import org.broadinstitute.dsde.firecloud.model.ModelSchema
 
 import scala.collection.immutable
-import scala.util.Try
+import scala.util.{Success, Try}
 
 import spray.json.DefaultJsonProtocol._
 import spray.json.JsValue
 
 object TSVFormatter {
 
-  def makeMembershipTsvString(entities: List[EntityWithType], entityType: String): String = {
+  def makeMembershipTsvString(entities: List[EntityWithType], entityType: String, collectionMemberType: String): String = {
     val headers: immutable.IndexedSeq[String] = immutable.IndexedSeq("membership:" + entityType + "_id", entityType.replace("_set", "_id"))
-    val rows: List[IndexedSeq[String]] = entities filter { _.entityType == entityType } flatMap { entity =>
-      entity.attributes.getOrElse(Map.empty) flatMap { m =>
-        m._2.convertTo[List[JsValue]] map { jsValue =>
-          val cellValue: String = jsValue match {
-            case x if Try(x.asJsObject.fields.contains("entityName")).isSuccess =>
-              cleanValue(x.asJsObject.fields.getOrElse("entityName", x))
-            case _ =>
-              ""
+    val rows: List[IndexedSeq[String]] = entities.
+      filter { _.entityType == entityType }.
```

Comment:
An alternate style (common in Rawls) is to make sure the `{` is usually on the first line.

`entities.filter {`